### PR TITLE
Update GUI

### PR DESCRIPTION
**Pull request by JonathanHoshi**
image

### CODE REVIEW

**Review by Ardentsoul — APPROVED** [review]
LGTM!

File: src/main/java/seedu/trackermon/logic/commands/CommandResult.java [hunk 1]
```diff
@@ -17,21 +17,33 @@
     /** The application should exit. */
     private final boolean exit;
 
+    /** The list will be modified **/
+    private final boolean modifyList;
```
Comment (lowkaiwei98) [inline src/main/java/seedu/trackermon/logic/commands/CommandResult.java:21]: Maybe can make it sound more boolean-like: isModifiedList? JustCurious how come we don't need this boolean before but not we need?

File: src/main/java/seedu/trackermon/logic/commands/CommandResult.java [hunk 2]
```diff
@@ -17,21 +17,33 @@
     /** The application should exit. */
     private final boolean exit;
```
Comment (lowkaiwei98) [inline src/main/java/seedu/trackermon/logic/commands/CommandResult.java:18]: Maybe can make it sound more boolean-like: isExit?

**Review by lowkaiwei98 — CHANGES_REQUESTED** [review]
Good job! Other than the few naming issues, LGTM!

File: src/main/java/seedu/trackermon/logic/commands/CommandResult.java (hunk 2 above)
Comment (JonathanHoshi) [reply src/main/java/seedu/trackermon/logic/commands/CommandResult.java]: Noted, it was originally like that from AB3's code. I'll changed it.

File: src/main/java/seedu/trackermon/logic/commands/CommandResult.java (hunk 1 above)
Comment (JonathanHoshi) [reply src/main/java/seedu/trackermon/logic/commands/CommandResult.java]: Noted!

**Pull request merged**